scripts/context.js

```diff
@@ -69,16 +69,21 @@ function singlePageView() {
       } else {
         contexts_dic[feature]()
         $(featureTabId).click(function(event) {
-          openContextFeature(event, featureId)
+          let openedFeature = openContextFeature(event, featureId) + '_tab'
+          chrome.storage.sync.set({openedFeature: openedFeature}, function() {
+          })
+        })
+        //Set the default feature to be opened
+        chrome.storage.sync.get(['openedFeature'], function(result) {
+          openedFeature = result.openedFeature
```

Comment:
Can we rename this variable ? And please initialise the variable properly. Thank you